openadmet_toolkit/structure/cofolding.py

```diff
@@ -27,17 +26,22 @@ def combine_seq_smiles_to_fasta(
 
     Parameters
     ----------
-    fasta : str
-        Fasta string
-    smiles : str
-        Smiles string
+    seqs : str
+        Fasta string or Smiles string
+    names: str
+        Name of protein or ligand
+    p_or_l:
+        Either "Protein" or "Ligand"
 
     Returns
     -------
     str
         Combined fasta string
     """
-    return f">protein|name={protein_name}\n{seq_string}\n>ligand|name={ligand_name}\n{smiles}"
+    to_return = ""
```

Comment:
Need to check that `p_or_l` is one of the expected values or raise and error

e.g 

```python
if not all(p_or_l in ["protein", "ligand"]:
    raise ValueError("unexpected tag, must be one of "protein", ligand"")
```